Assets/Scripts/Player/MovementController.cs

```diff
@@ -188,7 +188,31 @@ void Update()
                     currentMovement.y * Vector3.Cross(camera.transform.right, Vector3.up).normalized +
                     currentMovement.x * Vector3.Cross(Vector3.up, camera.transform.forward).normalized;
                 // Move (without gravity). Whenever we move we also readjust the player's direction to the direction they are running in.
-                characterController.Move(absoluteMovement * Time.deltaTime);
+
+                //temporary to keep us from being stuck in the air
+                Tile _currentTile = null;
+
+                try
+                {
+                    _currentTile = GameObject.Find(string.Format("tile_{0}_{1}", Mathf.Round(transform.position.x), Mathf.Round(transform.position.z))).GetComponent<Tile>();
+                }
+                catch
+                {
+
+                }
+                Vector3 _gravity;
+                if (_currentTile == null || _currentTile.turf == null)
+                {
+                    _gravity = new Vector3(0, 0, 0);
+                    characterAnimator.SetBool("Floating", true);
+                }
+                else
+                {
+                    _gravity = new Vector3(0, 9.8f, 0);
+                    characterAnimator.SetBool("Floating", false);
+                }
```

Comment:
This section can be shortened to something like:
```
bool floating = _currentTile == null || _currentTile.turf == null;
Vector3 _gravity = floating ? Vector3.zero : Vector3.down * 9.8f;
                
characterAnimator.SetBool("Floating", true);
```
To keep the code short for doing something this simple. For easy access you may also want to define gravity as a constant.